experiments/openebs/openebs-pool-disk-loss/vars.yml

```diff
@@ -0,0 +1,19 @@
+a_label: "{{ lookup('env','APP_LABEL') }}"
```

Comment:
As discussed with @ksatchit Can we put these variables inside the ansible_logic file otherwise, we have to do bulk changes

Reply:
As long as it work, we don't need it move the variables to the ansible_logic. Can you mention the reason with more info?

Reply:
cc @ksatchit

Reply:
And why do we need to change in all place? As this is specific to a particular experiment.

Reply:
- we need to change to maintain uniformity. If all experiments have different flow then it would be a little difficult for external users.

**Points to support- Not using a separate file**
- It will not increase any functionality, in fact,  it will increase an overhead to change the other experiments too.
- As no. of file increases, it will also increase the dependency among them and it becomes difficult to manage. If it is utils then it is good to have in a separate file to increase the re-usability but it is var_file which is different for all experiments so it will just increase dependency.
- This is a debatable topic having pro/cons both
cc: @ksatchit